### DISCUSSION TITLE
Anyone have commands and hooks they would be willing to share?

### DISCUSSION THREAD

**Discussion by HusseinT2000** [notaapp/talk#227] [Questions] [answered]
I don't know enough to be able to create hooks and commands so was wondering if anyone has created their own so far.

Also was thinking as a possible suggestion, especially if an API is made at some point some sort of store page where they would be collated.

**Comment by webwesen** [comment]
+1 for the curated repository in this org, populated through PRs

**Comment by astoilkov** [comment, marked as answer]
We are planning a `notaapp/extensions` repo that we will showcase a few extensions + allow people to contribute. What do you think about that?

**Reply by HusseinT2000** [reply]
> people

That sounds good, will it be so that extensions will be moderated or any extension can be shared and instantly used?

**Reply by astoilkov** [reply]
They will be moderated. We can list people's own repositories in the readme.

**Comment by davistech** [comment]
+1
